parquet-column/src/main/java/org/apache/parquet/schema/LogicalTypeAnnotation.java

```diff
@@ -1128,6 +1164,128 @@ public int hashCode() {
     }
   }
 
+  public static class GeometryLogicalTypeAnnotation extends LogicalTypeAnnotation {
+    private final String crs;
+
+    private GeometryLogicalTypeAnnotation(String crs) {
+      this.crs = crs;
+    }
+
+    @Override
+    @Deprecated
+    public OriginalType toOriginalType() {
+      return null;
+    }
+
+    @Override
+    public <T> Optional<T> accept(LogicalTypeAnnotationVisitor<T> logicalTypeAnnotationVisitor) {
+      return logicalTypeAnnotationVisitor.visit(this);
+    }
+
+    @Override
+    LogicalTypeToken getType() {
+      return LogicalTypeToken.GEOMETRY;
+    }
+
+    @Override
+    protected String typeParametersAsString() {
+      StringBuilder sb = new StringBuilder();
+      sb.append("(").append(crs != null && !crs.isEmpty() ? crs : "").append(")");
```

Comment:
Should we print an empty string when crs is unavailable?

Reply:
+1

It seems cleaner to write an empty string compared to "()".